composer.json

```diff
@@ -20,7 +20,8 @@
         "source": "https://github.com/ramsey/uuid"
     },
     "require": {
-        "php": ">=5.3.3"
+        "php": ">=5.3.3",
+        "paragonie/random_compat": "^2.0"
```

Comment:
Best to allow `^1.0|^2.0` otherwise will will conflict with just about everything atm.